service-front/module/Application/view/application/pages/driving_licence_number.twig

```diff
@@ -0,0 +1,80 @@
+<h1 class="govuk-heading-xl">UK Driving Licence</h1>
+<form method="POST" target="">
+    <h2 class="govuk-heading-m">Name on licence</h2>
+    <p class="govuk-body">
+        {{ details_data.Name }}
+    </p>
+    <h2 class="govuk-heading-m">Date of Birth</h2>
+    <p class="govuk-body">
+        {{ details_data.DOB }}
+    </p>
+    <div class="govuk-form-group {% if form.get('dln').messages %}govuk-form-group--error{% endif %}">
+        <h2 class="govuk-label-wrapper">
+            <label class="govuk-label govuk-label--m" for="dln">
+                Driving Licence Number
+            </label>
+        </h2>
+
+        <p id="dln-hint" class="govuk-hint">
+            A driving licence number is 16-digits. Example: MORGA657054SM9IJ
+        </p>
+
+        {% if form.get('dln').messages %}
+            <p id="dln-error" class="govuk-error-message">
+                <span class="govuk-visually-hidden">Error:</span>
+                {{ form.get('dln').messages | join(', ', ' and ') }}
+            </p>
+        {% endif %}
+
+        <input
+                class="govuk-input govuk-!-width-one-third {% if form.get('dln').messages %}govuk-input--error{% endif %}"
+                id="dln" name="dln" type="text" autocomplete="dln"
+                value="{{ form.get('dln').value }}"
+                aria-describedby="dln-hint {% if form.get('dln').messages %}dln-error{% endif %}"
+        />
+    </div>
+
+    <details class="govuk-details">
+        <summary class="govuk-details__summary">
+                <span class="govuk-details__summary-text">
+                  Where to find the Driving Licence Number
+                </span>
+        </summary>
+        <div class="govuk-details__text">
+            The driving licence number is found in section 5 of the details section for both paper and photo ID
+            licenses. Example:
+            <img src="{{ '/assets/driving_licence_sample.png' | basepath }}" alt="driving_licence_sample"/>
+        </div>
+    </details>
+
+    <fieldset class="govuk-fieldset">
```

Comment:
This fieldset should be wrapped in a `govuk-form-group` to give adequate spacing beneath it, and it should have `govuk-form-group--error` if the field has an error.

Also, the fieldset should get `aria-describedby="govuk-error-messag"` if the field has an error.